src/index.html

```diff
@@ -1,17 +1,150 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
   </body>
+  <form
+   action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+   method="get"
+   class="form"
+   autocomplete="off"
+  >
+    <fieldset class="block">
+      <legend>Personal information</legend>
+      <label class="form-field">
+        Surname:
+        <input
+          type="text"
+          name="surname"
+          required
+        >
+      </label>
+      <label class="form-field">
```

Comment:
Add blank lines between elements in all places
```suggestion
      </label>
      
      <label class="form-field">
```